apps/site/pages/docs/go-live/3-integrating-the-the-vtex-login.mdx

```diff
@@ -13,4 +13,45 @@ import { Callout, Tab, Tabs } from 'nextra-theme-docs'
 
 <Callout type="warning" emoji="⚠️">
 This documentation is currently under development.
-</Callout>
\ No newline at end of file
+</Callout>
+
+In this guide, you'll learn how to integrate the **VTEX Login** with your **FastStore** project.
+
+The VTEX Login is provided by the `vtexcommercestable` environment and uses a subdomain. Therefore, to successfully integrate the VTEX Login with your FastStore project, you must add a redirect back to the FastStore environment and set up the necessary auth cookies to authenticate client requests and maintain session information. This way, once shoppers finish logging in to your store, they can be redirected back to your website's main domain.
+
+---
+
+## Before you start
+
+- Ensure you have already [configured the custom domain name of your FastStore website](/docs/go-live/2-configuring-external-dns).
+- Stores that currently use Portal (Legacy) technology are required to update their `vtex.edition-business@0.x` to `vtex.edition-store@5.x,`  and to migrate their secure domain to VTEX IO. To make these changes, please open a support ticket with the [VTEX support team](https://help.vtex.com/support) and request that the [Edition App](https://developers.vtex.com/docs/guides/vtex-io-documentation-edition-app) installed in your account be updated and that your secure domain be migrated from Portal (Legacy) to VTEX IO.
```

Comment:
Is there any other information to highlight for Legacy stores migrating to FastStore?

Reply:
@Mariana-Caetano I'm trying to get more info from other stores that have done this migration and will send it your way about what they've done.

Reply:
@gvc any news? 👀

Reply:
@gvc I've made some changes based on the doc you shared about the legacy store. The changes you can check here: https://github.com/vtex/faststore/pull/1807/files#diff-dc9b1b7354ff76188e7358360b4f3b6d8db9871931fd7ce6dd3473ee396ab265R23-R33

Do you think we can move forward and merge this PR?